apps/docs/src/content/reference/extras/trackball-controls.mdx

```diff
@@ -0,0 +1,48 @@
+---
+{
+  order: 9.1,
+  'category': '@threlte/extras',
+  sourcePath: 'packages/extras/src/lib/components/controls/TrackballControls/TrackballControls.svelte',
+  'name': '<TrackballControls>',
+  'type': 'component',
+  'componentSignature':
+    {
+      extends:
+        {
+          type: 'TrackballControls',
+          url: 'https://threejs.org/docs/#examples/controls/TrackballControls'
+        }
+    }
+}
+---
+
+`<TrackballControls>` TrackballControls is similar to OrbitControls. However, it does not maintain a constant camera up vector. That means if the camera orbits over the “north” and “south” poles, it does not flip to stay "right side up"
```

Comment:
Could you add a little comment about the usecases for `<TrackballControls>` over `<OrbitControls>`?

Reply:
done!

Reply:
Great writing! Super clear. 

Only comment is that links to components should include the both code styling and angled braces, e.g. [see links in the `<Sheet>` docs](https://threlte-mytcm6ro7-threlte.vercel.app/docs/reference/theatre/sheet). Same for the link you added in `<OrbitControls>`.

Reply:
done!